### ISSUE TITLE
liquibase 3.10.2 fixes bug with mariadb

### ISSUE THREAD

**Issue by klenkes74** [quarkusio/quarkus#11762]
**Describe the bug**
Up to version 3.10.1 there is a liquibase bug with mariadb that prevents it from running (https://liquibase.jira.com/browse/CORE-3457). It is said to be fixed in version 3.10.2.

**Expected behavior**
Liquibase runs without exception on mariadb.

**Actual behavior**
Liquibase throws an exception

```
Caused by: liquibase.exception.DatabaseException: liquibase.command.CommandExecutionException: liquibase.exception.UnexpectedLiquibaseException: Don't know how to query for sequences on db @ jdbc:mariadb://host:3306/db?autoReconnect=true&useResetConnection=true (Default Schema: db)
```

**To Reproduce**
1. Add liquibase to the project
2. add even an empty db/changelog.xml
3. run quarkusDev

**Configuration**
```properties
quarkus.liquibase.migrate-at-start = true
quarkus.liquibase.validate-on-migrate = false
quarkus.liquibase.clean-at-start = true
quarkus.liquibase.change-log = db/changeLog.xml

%dev.quarkus.liquibase.change-log=db/changeLog-dev.xml
%dev.quarkus.datasource.db-kind = mariadb
%dev.quarkus.datasource.username = db
%dev.quarkus.datasource.password = <password>
%dev.quarkus.datasource.jdbc.url = jdbc:mariadb://hostname:3306/db?autoReconnect=true&useResetConnection=true
```

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
```
Linux rlichti.remote.csb 4.18.0-80.11.2.el8_0.x86_64 #1 SMP Sun Sep 15 11:24:21 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```
 - Output of `java -version`:
```
OpenJDK Runtime Environment GraalVM CE 20.0.0 (build 11.0.6+9-jvmci-20.0-b02)
OpenJDK 64-Bit Server VM GraalVM CE 20.0.0 (build 11.0.6+9-jvmci-20.0-b02, mixed mode, sharing)
```
 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: 1.7.1.Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
```
------------------------------------------------------------
Gradle 6.6.1
------------------------------------------------------------

Build time:   2020-08-25 16:29:12 UTC
Revision:     f2d1fb54a951d8b11d25748e4711bec8d128d7e3

Kotlin:       1.3.72
Groovy:       2.5.12
Ant:          Apache Ant(TM) version 1.10.8 compiled on May 10 2020
JVM:          11.0.6 (Oracle Corporation 11.0.6+9-jvmci-20.0-b02)
OS:           Linux 4.18.0-80.11.2.el8_0.x86_64 amd64
```

**Additional context**
The bug in liquibase is acknowledged and closed in https://liquibase.jira.com/browse/CORE-3457

**Comment by klenkes74** [comment]
I wanted to fix it - but in master liquibase is already set to liquibase 4.0.0 - so the next release will fix the problem It can be closed.

**Comment by gsmet** [comment]
Let's fix it in the 1.7 branch.

**Comment by jaikiran** [comment]
https://github.com/quarkusio/quarkus/pull/11776 has been merged.
